### DISCUSSION TITLE
Write cache to destination folder

### DISCUSSION THREAD

**Discussion by pchmykh** [NickvisionApps/Parabolic#692] [Q&A]
Let's discuss is it good idea to download all the files through cache folder, located at home directory.
If you have dedicated drive to downloads, it leads to unnecessary usage of home containing disk.

**Comment by nlogozzo** [comment]
Well the cache is just that, cache. It's temporary files, that yes get stored on the home drive, but they get cleaned up by Parabolic on application close/startup. So it's not unnecessary usage since again they get deleted eventually.

**Comment by nlogozzo** [comment]
BTW, Parabolic uses the directory configured by XDG_CACHE_DIR . So you can always change that and point it to the directory that you'd like GNOME apps to use for cache.
